mathics/builtin/inference.py

```diff
@@ -298,10 +294,8 @@ def get_assumption_rules_dispatch(evaluation):
 
         if value:
             symbol_value = SymbolTrue
-            symbol_negate_value = SymbolFalse
```

Comment:
Sure that these are not necesary? I do not remember exactly why I put these lines...

Reply:
In that case, go ahead...